server/jvm/{{appName}}-distribution/build.gradle.kts

```diff
@@ -69,6 +69,10 @@ tasks {
         archiveBaseName.set("genesisproduct-{{appName}}")
         archiveClassifier.set("bin")
         archiveExtension.set("zip")
+        inputs.files(rootProject.getTasksByName("copyDependencies", true))
+    }
+    distTar {
+        inputs.files(rootProject.getTasksByName("copyDependencies", true))
```

Comment:
do we care about distTar at all? we only distribute zip files afaik

Reply:
I think, with the enhanced strictness in latest gradle version, it complains about it. If we don't care about the task then can we disable it?

Reply:
Disabled it.